src/sentry/static/sentry/app/actionCreators/teams.jsx

```diff
@@ -93,3 +95,34 @@ export function leaveTeam(api, params, options) {
     },
   });
 }
+
+export function removeTeam(api, params, options) {
+  TeamActions.removeTeam(params.teamId);
+
+  return api
+    .requestPromise(`/teams/${params.orgId}/${params.teamId}/`, {
+      method: 'DELETE',
+    })
+    .then(
+      data => {
+        TeamActions.removeTeamSuccess(params.teamId, data);
+        addSuccessMessage(
```

Comment:
i know this has been gone over a few times, but these alerts are still easy to miss imo